Makefile

```diff
@@ -3,6 +3,7 @@ SRCS_JS_FROM_TS =
 
 SRCS_TS += index.ts
 SRCS_TS += $(shell find lib -type f -name '*.ts')
+SRCS_TS += $(shell find test -type f -name '*.ts')
```

Comment:
@ericvw Sure. I will strip out the `Makefile` and changes related to unit tests into a separate PR. At least I am more confident for the `HistoricalBufferManager` now with the tests. :smile: